test/integration/features/addon-xpaas.feature

```diff
@@ -19,18 +19,19 @@ which are then available in OpenShift to the user.
 
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
-    When executing "oc new-app <template-name>" succeeds
+    When executing "oc new-project <project-name>" succeeds
+    Then executing "oc new-app <template-name>" succeeds
      And executing "oc set probe dc/<service-name> --readiness --get-url=http://:8080<http-endpoint>" succeeds
      And service "<service-name>" rollout successfully within "1200" seconds
-    Then with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" contains "<expected-hello>"
-     And with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" is equal to "200"
-     And executing "oc delete all --all" succeeds
+     And with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "<project-name>" contains "<expected-hello>"
```

Comment:
Here I would suggest `Then` as in this moment all preparation steps are done - when I created project and created app from template and set probes and waited for the rollout, then I can do something: check the site, check that and later delete it all.

Reply:
@agajdosi let me know if this is fine for you
```   
   Given Minishift has state "Running"
    When ...
     And ...
     And ...
     And ...
    Then ...
     And ...
     And ...
```